sdk/keyvault/Azure.ResourceManager.KeyVault/src/autorest.md

```diff
@@ -90,6 +90,7 @@ directive:
       $.VaultCreateOrUpdateParameters.properties.location['x-ms-format'] = 'azure-location';
       $.VaultAccessPolicyParameters.properties.location['x-ms-format'] = 'azure-location';
       $.Vault.properties.location['x-ms-format'] = 'azure-location';
+      $.Vault['x-csharp-usage'] = 'model,input,output';
```

Comment:
Shall we try ` $.Vault['x-csharp-usage'] = 'model,output';` as VaultData is not used in requests?

Reply:
Add `input` is used to make it's properties type match with `TrackedResourceData`